test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/FilterAndOrderByFunctionalTests.cs

```diff
@@ -2138,17 +2138,18 @@ public void FilterWithInOperationWithMalformCollection(string collection)
         [Fact]
         public void FilterWithInOperationWithParensStringCollection_EscapedSingleQuote()
         {
-            FilterClause filter = ParseFilter("SSN in ('a''bc','''def','xyz''')", HardCodedTestModel.TestModel, HardCodedTestModel.GetPersonType());
+            FilterClause filter = ParseFilter("SSN in ('a''bc','''def','ghi''','xyz''')", HardCodedTestModel.TestModel, HardCodedTestModel.GetPersonType());

```

Comment:
what's the difference between 'ghi''' and 'xyz''''? I mean It looks like they are the same scenario. Does it mean double single quotes at the end or in the middle?